libs/langchain/langchain/llms/minimax.py

```diff
@@ -87,6 +87,11 @@ def validate_environment(cls, values: Dict) -> Dict:
             "MINIMAX_API_HOST",
             default="https://api.minimax.chat",
         )
+        cls._client = _MinimaxEndpointClient(
```

Comment:
i think slightly better to do 
```suggestion
        values["_client"] = _MinimaxEndpointClient(
```

Reply:
agree